Makefile

```diff
@@ -146,7 +146,7 @@ prepare-wget-macos:
 prepare-tests-macos: prepare-wget-macos prepare-tests-files
 	brew install graphviz || true
 
-prepare-tests-ubuntu: prepare-tests-files
+prepare-tests-ubuntu: prepare-tests-files install-full
```

Comment:
doing this means that `make install-full` runs twice in the CI, right? even if the second time it doesn't do much. Is it inconvenient?

Also, we mention `prepare-tests-ubuntu` in the README. Is there something we should update there?

Reply:
Yep, we have to run all the extras in full in the CI twice, because poetry introduced a change in behaviour in the new minor (whenever you install a specific extra only, e.g. spacy, it will uninstall all other extras from the environment). More details in this [thread](https://rasa-hq.slack.com/archives/C01HHMR4X8S/p1667924056444669).

Reply:
I don't think we need to update the README.

Reply:
👍🏻 can we write a comment above the makefile target to explain why it runs twice? (basically what you just wrote :))